### PR TITLE
Add protocol and protocol_version variables

### LINKED ISSUE #215: Missing protocol and protocol_version variables for alb_ingress module
Found a bug? Maybe our [Slack Community](https://slack.cloudposse.com) can help. 

[Slack Community](https://slack.cloudposse.com)

## Describe the Bug
This [module](https://github.com/cloudposse/terraform-aws-ecs-web-app/blob/master/main.tf#L23) does not support variables for setting `protocol` or `protocol_version`, resulting in the hardcoded setting of `HTTP` and `HTTP1`. This prevents use of HTTPS and HTTP/2 for the default target group created by the `alb_ingress` module.

## Expected Behavior
Ability to pass in the alb ingress `protocol` and `protocol_version` variables. Want to use this module for gRPC over HTTP2, which requires HTTPS for the ALB.

## Steps to Reproduce
2. Run '....'
3. Enter '....'

## Screenshots
If applicable, add screenshots or logs to help explain your problem.

## Environment (please complete the following information):

Anything that will help us triage the bug will help. Here are some ideas:
 - OS: [e.g. Linux, OSX, WSL, etc]
 - Version [e.g. 10.15]

### PR DESCRIPTION
**Pull request by zaksamalik**
## what
* adds `alb_ingress_protocol` and `alb_ingress_protocol_version` variables for passing to alb_ingress module.

## why
* enables use of HTTPS protocol with HTTP2 or GRPC protocol versions.

## references
* Closes #215 
* https://github.com/cloudposse/terraform-aws-alb-ingress/blob/master/variables.tf#L121

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -470,6 +470,18 @@ variable "alb_ingress_listener_authenticated_priority" {
...
+variable "alb_ingress_protocol" {
+  type        = string
+  default     = "HTTP"
+  description = "The protocol for the created ALB target group (if target_group_arn is not set). One of `HTTP`, `HTTPS`. Defaults to `HTTP`."
+}
+
+variable "alb_ingress_protocol_version" {
+  type        = string
+  default     = "HTTP1"
+  description = "he protocol version. One of `HTTP1`, `HTTP2`, `GRPC`. Only applicable when protocol is HTTP or HTTPS. Specify GRPC to send requests to targets using gRPC. Specify HTTP2 to send requests to targets using HTTP/2. The default is `HTTP1`, which sends requests to targets using HTTP/1.1"
```
Comment (nitrocode) [inline variables.tf:482]:
```suggestion
  description = "The protocol version. One of `HTTP1`, `HTTP2`, `GRPC`. Only applicable when protocol is HTTP or HTTPS. Specify GRPC to send requests to targets using gRPC. Specify HTTP2 to send requests to targets using HTTP/2. The default is `HTTP1`, which sends requests to targets using HTTP/1.1"
```

Comment (nitrocode) [reply variables.tf:482]: @zaksamalik

Comment (nitrocode) [reply variables.tf:482]: Please also run `make pr/auto-format`

Comment (zaksamalik) [reply variables.tf]: @nitrocode updated and formatted. Also just want to call out this bumps the pinned version for `alb-ingress` module to `0.25.1` to support the `protocol_version` variable.

**Review by nitrocode — APPROVED** [review]

**Pull request merged**